src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -578,69 +578,36 @@
   - name: Create a task without the source location.
     text: >
         az acr task create -n hello-world -r MyRegistry --cmd '$Registry/myimage' -c /dev/null
-  - name: Create a task without the source location and with a timer trigger that runs the image `MyImage` at the top of every hour using the default trigger name.
-    text: >
-        az acr task create -n hello-world -r MyRegistry --cmd '$Registry/myimage' -c /dev/null --schedule "0 */1 * * *"
-  - name: Create a task with the definition from the standard input. Either 'Ctrl + Z'(Windows) or 'Ctrl + D'(Linux) terminates the input stream.
-    text: >
-        az acr task create -n hello-world -r MyRegistry -f - -c /dev/null
-  - name: Create a task with the definition from the pipe.
```

Comment:
keep the pipe example?

Reply:
From my experience with Linux users as long as you say it takes the info from standard input they assume you can pipe it. But I was also conflicted as if to remove it or not. I do think more people will use the pipe, but the input stream seemed more educational. 
It's up to you. However, I do think it would be better to have only one Standard input example. In case you want both, I could also mention in the description that it can be piped, put the pipe example and mention in can be taken from the input stream, or just add both examples within the same text section.